nixos/modules/programs/ente-auth.nix

```
@@ -0,0 +1,30 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.programs.ente-auth;
+in
+{
+  options.programs.ente-auth = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      example = true;
+      description = ''
+        Whether to enable Ente Auth
+      '';
+    };
```

There's already a wrapper for such options:
```suggestion
    enable = lib.mkEnableOption "Ente Auth";
```